### PR TITLE
Adds more vaurca loadout items

### PR DESCRIPTION
**Pull request by Alberyk**
Added some vaurca masks at the request of their lore dev to the loadout.

### CODE REVIEW

**Comment by BotBOREALIS** [comment]
Icons with diff:
<details><summary>icons/obj/vaurca_items.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
skrell_mask||skrell_mask|Created
skrell_mask_ma||skrell_mask_ma|Created
skrell_mask_ma[1]||![skrell_mask_ma[1]](http://images.aurorastation.org/images/ac265f3387d5b2d7a3d02eb171b8bcad.png)|Created
skrell_mask_ma[2]||![skrell_mask_ma[2]](http://images.aurorastation.org/images/a3df1eac38e7dcacd58092a46ac4bd32.png)|Created
skrell_mask_ma[3]||![skrell_mask_ma[3]](http://images.aurorastation.org/images/67f6fc72869a3554a8a2668b47b25d1f.png)|Created
human_mask||human_mask|Created
human_mask_ma||human_mask_ma|Created
human_mask_ma[1]||![human_mask_ma[1]](http://images.aurorastation.org/images/13002e953397e7b89ecc59af8450f7bb.png)|Created
human_mask_ma[2]||![human_mask_ma[2]](http://images.aurorastation.org/images/06e3f3f4a575039bd034c593c7b2682c.png)|Created
human_mask_ma[3]||![human_mask_ma[3]](http://images.aurorastation.org/images/41cd569a048d7c73b8bcf98418861639.png)|Created
</details>

**Comment by ParadoxSpace** [comment]
post images

**Comment by Alberyk** [comment]
image

File: code/modules/client/preference_setup/loadout/loadout_xeno.dm [hunk 1]
```diff
@@ -99,12 +99,20 @@
...
 	cost = 1
 	whitelisted = list("Vaurca Worker", "Vaurca Warrior",)
 	sort_category = "Xenowear"
 
+/datum/gear/mask/vaurca/New()
+	..()
+	var/masks = list()
+	masks["mandible garment"] = /obj/item/clothing/mask/breath/vaurca
+	masks["human expression mask"] = /obj/item/clothing/mask/breath/vaurca/expression
+	masks["vaurca expression mask"] = /obj/item/clothing/mask/breath/vaurca/expression/skrell
```
Comment (LordFowl) [inline code/modules/client/preference_setup/loadout/loadout_xeno.dm]: Don't you mean masks["skrell expression mask"]? An expression mask of an uenxpressive alien seems expressly unexpressing.

File: code/game/objects/items/weapons/vaurca_items.dm [hunk 2]
```diff
@@ -13,6 +13,18 @@
 	user << "This mask is too tight to adjust."
 	return
 
+/obj/item/clothing/mask/breath/vaurca/expression
+	name = "human expression mask"
+	desc = "A mask that allows emotively challenged aliens  to convey facial expressions. This one depicts a human."
```
Comment (LordFowl) [inline code/game/objects/items/weapons/vaurca_items.dm]: Double space between "aliens" and "to".

File: code/game/objects/items/weapons/vaurca_items.dm [hunk 3]
```diff
@@ -13,6 +13,18 @@
...
+	icon_state = "human_mask"
+	item_state = "human_mask"
+
+/obj/item/clothing/mask/breath/vaurca/expression/skrell
+	name = "skrell expression mask"
+	desc = "A mask that allows emotively challenged aliens  to convey facial expressions. This one depicts a skrell."
```
Comment (LordFowl) [inline code/game/objects/items/weapons/vaurca_items.dm]: Double space between "aliens" and "to".

**Comment by Butterrobber202** [comment]
This is a good change

**Pull request merged**